src/System.Management.Automation/engine/CommandCompletion/CompletionCompleters.cs

```diff
@@ -4404,6 +4404,11 @@ private struct SHARE_INFO_1
         private const int ERROR_MORE_DATA = 234;
         private const int STYPE_DISKTREE = 0;
         private const int STYPE_MASK = 0x000000FF;
+        private static System.IO.EnumerationOptions _enumerationOptions = new System.IO.EnumerationOptions
+        {
+            MatchCasing = MatchCasing.CaseInsensitive,
+            AttributesToSkip = 0
```

Comment:
You added `AttributesToSkip = 0`. Please clarify why we need this?

Reply:
I'll add a comment